Src/WitsmlExplorer.Api/Routes.cs

```diff
@@ -95,7 +95,7 @@ public static void ConfigureApi(this WebApplication app, IConfiguration configur
             app.MapGet("/jobs/userjobinfo/{jobId}", JobHandler.GetUserJobInfo, useOAuth2);
             app.MapGet("/jobs/alljobinfos", JobHandler.GetAllJobInfos, useOAuth2, AuthorizationPolicyRoles.ADMINORDEVELOPER);
             app.MapPost("/jobs/cancel/{jobId}", JobHandler.CancelJob, useOAuth2);
-            app.MapGet("/jobs/getreportitems/{jobId}", JobHandler.GetReportItems, useOAuth2);
+            app.MapGet("/jobs/getreport/{jobId}", JobHandler.GetReport, useOAuth2);
```

Comment:
Rename `"/jobs/getreport/{jobId}"` to `"/jobs/report/{jobId}"`

Reply:
fixed